crates/router/src/connector/paybox/transformers.rs

```diff
@@ -50,6 +55,13 @@ type Error = error_stack::Report<errors::ConnectorError>;
 pub enum PayboxPaymentsRequest {
     Card(PaymentsRequest),
     CardThreeDs(ThreeDSPaymentsRequest),
+    Mandate(MandatePaymentRequest),
+}
+
+#[derive(Debug, Serialize)]
+pub struct CardMandateInfo {
+    pub card_exp_month: Secret<String>,
+    pub card_exp_year: Secret<String>,
```

Comment:
You can also use `CardExpirationMonth` and `CardExpirationMonth` from the `cards` crate here, instead of `Secret<String>`. They also allow formatting the year with 2/4 digits, as required.